### DISCUSSION TITLE
Can anyone tell me why this problem occurs after the update?

### DISCUSSION THREAD

**Discussion by L-J-Yinph** [saicaca/fuwari#248] [Q&A]
![image](https://github.com/user-attachments/assets/e908a38f-d469-4d4c-b59d-55fb61489837)
How can I solve this situation? Who can help me?

**Comment by saicaca** [comment]
It might be because your local dependencies haven't been updated; running `pnpm install` again might solve the issue.

**Reply by L-J-Yinph** [reply]
OK, I've solved it now. Thank you for your reply.
